subsys/debug/symtab/symtab.c

```diff
@@ -10,16 +10,16 @@
 #include <zephyr/debug/symtab.h>
 #include <zephyr/shell/shell.h>
 
-const struct symtab_info *const symtab_get(void)
+const struct symtab_info *symtab_get(void)
 {
 	extern const struct symtab_info z_symtab;
 
 	return &z_symtab;
 }
 
-const char *const symtab_find_symbol_name(uintptr_t addr, uint32_t *offset)
+const char *symtab_find_symbol_name(uintptr_t addr, uint32_t *offset)
 {
-	const struct symtab_info *const symtab = symtab_get();
```

Comment:
I don't think you need to remove this, as this is marking that the pointer should not be modified.

Reply:
done